### PR TITLE
chore(test): migrate scoped-slot-text

### PR DESCRIPTION
**Pull request by christian-bromann**
Migrate scoped-slot-text

### CODE REVIEW

File: test/karma/test-app/scoped-slot-text-with-sibling/karma.spec.ts [hunk 1]
```diff
@@ -15,9 +15,9 @@ describe('scoped-slot-text-with-sibling', () => {
...
    * @returns the custom element
    */
-  function getCmpLabel(): HTMLCmpLabelElement {
+  function getCmpLabel(): HTMLCmpLabelWithSlotSiblingElement {
     const customElementSelector = 'cmp-label-with-slot-sibling';
-    const cmpLabel: HTMLCmpLabelElement = app.querySelector(customElementSelector);
+    const cmpLabel: HTMLCmpLabelWithSlotSiblingElement = app.querySelector(customElementSelector);
     if (!cmpLabel) {
       fail(`Unable to find element using query selector '${customElementSelector}'`);
     }
```
Comment (alicewriteswrongs) [inline test/karma/test-app/scoped-slot-text-with-sibling/karma.spec.ts:23]: did you mean to include these edits to a different karma test?

Comment (christian-bromann) [reply test/karma/test-app/scoped-slot-text-with-sibling/karma.spec.ts:23]: Nope, for some reason this Karma test was using types generated by the directory I migrated. I updated the test to use the right class.

Comment (rwaskiewicz) [reply test/karma/test-app/scoped-slot-text-with-sibling/karma.spec.ts:23]: I `blame` me 😆 
Screenshot 2024-03-19 at 8 36 23 AM

Can we add a note in the final commit message about this? A lot of these `scoped-` prefixed tests are look pretty similar in name. I think it'd be worth a clarification note for another 2.5 years from now 😉

**Review by rwaskiewicz — CHANGES_REQUESTED** [review]
One small ask regarding that type change RE: adding it to the git commit msg

**Comment by christian-bromann** [comment]
Thanks for the review. @rwaskiewicz added a note to the commit message.

**Review by rwaskiewicz — APPROVED** [review]
Thanks!

**Review by alicewriteswrongs — APPROVED** [review]

**Pull request merged**